src/slinky/slinky.js

```diff
@@ -188,25 +190,8 @@ class Slinky extends React.Component {
             }
         };
 
-        const slinkySections = _.map(sections, (section, num) => {
-            const { header, content } = section;
-
-            const boundHeaderClick = this.handleHeaderClick.bind(this, num);
-
-            return (
-                <section className="slinky-section" style={sectionStyle} key={num}>
-                    <header
-                      className="slinky-header"
-                      style={{ overflow: 'hidden', cursor: 'pointer', ...headerStyle }}
-                      onClick={boundHeaderClick}
-                    >
-                        {header}
-                    </header>
-                    {content}
-                </section>
-            );
-        });
-
+        const slinkySections = renderSlinkySections({ sections, sectionStyle, headerStyle, handleHeaderClick: this.handleHeaderClick });
```

Comment:
Good call to break the section out into its own component! This line is hitting the max length rule though, so my only suggestion is to break the line up.